packages/SwingSet/test/test-kernel.js

```diff
@@ -866,6 +866,7 @@ test('transcript', async t => {
         response: null,
       },
     ],
+    crankNumber: 1,
```

Comment:
can we find an additional place to test that `crankNumber` gets incremented? somewhere we can assert that it equals at least "2".

Reply:
Actually, the fact that `crankNumber` is 1 shows that it was incremented.  Otherwise it would be 0.